openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -178,7 +184,7 @@ def test_get_form_failing_auth_url_transport(self):
         """Ensure GETing form over OAuth with correct client credentials in query fails when DEBUG is False"""
         query = urlencode({'access_token': self.access_token.token})
         response = self.csrf_client.get('/oauth2-test/?%s' % query)
-        self.assertIn(response.status_code, (status.HTTP_401_UNAUTHORIZED, status.HTTP_403_FORBIDDEN))
```

Comment:
I don't think they ever did.  It looks the like the test writer just lacked conviction on what was getting returned.